src/components/teamPhotos.js

```diff
@@ -53,18 +58,99 @@ const ProfileRole = styled.span`
 `
 
 const ProfileImg = styled(Img)`
-  border-radius: 50%;
+  transition: filter ease-in 200ms;
+`
+
+const ProfileImgWrapper = styled.div`
+  position: relative;
+
+  @media (min-width: 1000px) {
+    &:hover .profileImg {
+      filter: blur(1.5px);
+    }
+    &:hover .web {
+      display: block;
+    }
+  }
+`
+
+const ProfileIcon = ({ dest, icon }) => {
+  return (
+    <a href={dest} target="_blank">
+      <MediaQuery maxDeviceWidth={999}>
+        <StyledIcon icon={icon} color={SB_NAVY} />
+      </MediaQuery>
+      <MediaQuery minDeviceWidth={1000}>
+        <StyledIcon icon={icon} color={"white"} />
+      </MediaQuery>
+    </a>
+  )
+}
+
+const StyledIcon = styled(FontAwesomeIcon)`
+  color: ${props => props.color};
+  margin: 0.3rem 0.5rem 0rem 0.5rem;
+
+  @media (max-width: 999px) {
+    font-size: 1rem;
+  }
+  @media (min-width: 1000px) {
+    font-size: 1.5rem;
+  }
+`
+
+const IconWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  /* same as width of images coming in from GraphQL */
+  max-width: 150px;
+  margin: auto;
+
+  @media (max-width: 999px) {
```

Comment:
For this rule, you can actually leave out the entire media query. Since the other media query is `min-width: 1000px`, it won't get taken into effect until the screen reaches that width. This means that any default rules (outside of a media query) you write will be in effect until that point (<1000px).